recipe/meta.yaml

```diff
@@ -14,7 +14,7 @@ build:
 
 requirements:
   run:
-    - cuda-nsight 12.0.78
+    - cuda-nsight 12.0.78               # [not aarch64]
```

Comment:
Since `cuda-nsight` is [skipped on `linux_aarch64`]( https://github.com/conda-forge/cuda-nsight-feedstock/blob/567608cac8247e243e5a775ac2a5c6f1762f69ea/recipe/meta.yaml#L19 ), drop this dependency on `linux_aarch64`. This should fix [the CI failure on `linux_aarch64`]( https://app.travis-ci.com/github/conda-forge/cuda-visual-tools-feedstock/jobs/611359660#L658 )

Reply:
Also +1 here, thanks!